apis/v1alpha2/shared_types.go

```diff
@@ -399,3 +399,86 @@ type AnnotationKey string
 // +kubebuilder:validation:MinLength=0
 // +kubebuilder:validation:MaxLength=4096
 type AnnotationValue string
+
+// AddressRouteMatches defines AddressMatch rules for inbound traffic according to
+// source and/or destination address of a packet or connection.
+type AddressRouteMatches struct {
+	// SourceAddresses indicates the originating (source) network
+	// addresses which are valid for routing traffic.
+	//
+	// Support: Core
```

Comment:
Nit: I think this should probably be "Extended" to start, and we can potentially graduate to "Core" in the future if we see wide enough support.

Reply:
updated in bb3672a1517853dbbfec47b7c02a5b784d376bc1 :+1: